test/README.md

```diff
@@ -20,6 +20,10 @@ Some examples of test harnesses using these tests, from the source repository:
 - [Data model stringify tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/data-model/stringify.test.ts)
 - [Formatting tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/messageformat.test.ts)
 
+A [JSON schema](./schemas/) is included for the test files in this repository.
+
+For users of Visual Studio Code, a [settings file](./.vscode/settings.json) is included that enables schema validation while editing the test files.
```

Comment:
Line breaks

Reply:
This was added by #767, after which I merged `main` to this branch.